packages/frontend/src/components/Election/Results/ViewElectionResults.tsx

```diff
@@ -3,20 +3,24 @@ import { useParams } from "react-router";
 import Results from './Results';
 import Box from '@mui/material/Box';
 import { Paper, Typography } from "@mui/material";
-import { DetailExpanderGroup, useSubstitutedTranslation } from '../../util';
+import { useSubstitutedTranslation } from '../../util';
 import { useGetResults } from '../../../hooks/useAPI';
 import useElection from '../../ElectionContextProvider';
 import DraftWarning from '../DraftWarning';
+import { StyledButton } from '~/components/styles';
+import ShareButton from '../ShareButton';
+import { DownloadCSV } from './DowloadCSV';
+import { useGetBallots } from '~/hooks/useAPI';
 
 const ViewElectionResults = () => {
     
     const { election } = useElection()
     
     const { data, isPending, error, makeRequest: getResults } = useGetResults(election.election_id)
     useEffect(() => { getResults() }, [])
-
+	const { data: ballotsData, isPending: isPendingBallots, error: ballotsError, makeRequest: fetchBallots } = useGetBallots(election.election_id)
```

Comment:
Can you double check the permissions on the useGetBallots api in the backend?

I expect the API is only available to the election admin, but we want that to change based on the public_download setting

Reply:
Do we need an extra setting or should it be based on public_results?

Reply:
We elaborated further in slack: https://starvoting.slack.com/archives/C01EBAT283H/p1726845204279549